docs/installation.md

```diff
@@ -0,0 +1,58 @@
+# Actions Configuration
+
+## Enable GitHub Actions to generate Pull Requests
+
+In the admin permissions for the organization or repository enable the option to [Allow GitHub Actions to create and approve pull requests](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#preventing-github-actions-from-creating-or-approving-pull-requests).
+This is needed in order to allow the GitHub action to create and open the PRs.
+
+## Disable the automatic deletion of branches after pull requests are merged
+
+The action relies on the branch that opens the PR to remain in place so that the subsequent merges can still occur. The option for [Automatically delete head branches](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-the-automatic-deletion-of-branches) must be deselected in order for the action to work properly.
+
+## Leveraging an Additional Merge Token to enable commits to protected branches
+
+There are cases where you may want to use a branch protection rule for branch that needs to use the Cascade Merge. In those cases you'll need to use an additional variable named `MERGE_TOKEN` that has elevated access to the repository. The pre-existing `GITHUB_TOKEN` doesn't have enough permissions to bypass the restriction.
+
+Therefore you'll need to create a Machine User with Admin access or a [custom GitHub App](https://docs.github.com/en/developers/apps/building-github-apps/creating-a-github-app) that only needs read/write for the contents permissions. Than either of these can be used in the selected option to [Allow specified actors to bypass required pull requests](https://github.blog/changelog/2021-11-19-allow-bypassing-required-pull-requests/).
```

Comment:
```suggestion
Therefore you'll need to create a Machine User with Admin access or a [custom GitHub App](https://docs.github.com/en/developers/apps/building-github-apps/creating-a-github-app) that only needs read/write for the contents permissions. Then either of these can be used in the selected option to [Allow specified actors to bypass required pull requests](https://github.blog/changelog/2021-11-19-allow-bypassing-required-pull-requests/).
```

Reply:
I think then is right? lol don't trust me on this but

Reply:
Just fixed it nice catch